### DISCUSSION TITLE
Report an extension issue here

### DISCUSSION THREAD

**Discussion by jeevan-lal** [jeevan-lal/Excel-Data-Filler-V4#18] [Problems]
We need problems..

**Comment by ryzugai** [comment]
Microsoft Cursor Engine error '80040e21'

Multiple-step operation generated errors. Check each status value.

/iSMP/LIndustri/Frm_LI_DafSykt.asp, line 665

My problem is the form's final submission is the save button, however, the error occurs like the above message

**Reply by jeevan-lal** [reply]
Send the pic of where this error is occurring.

**Comment by ryzugai** [comment]
![Screenshot 2025-02-15 213350](https://github.com/user-attachments/assets/41eb15b9-149e-4a91-b538-36a2fadc94b9)
![Screenshot 2025-02-15 213341](https://github.com/user-attachments/assets/1a484ffc-dc87-4150-9169-00c10e57cf82)

**Reply by jeevan-lal** [reply]
> there was an issue during a multi-step database operation, such as fetching or updating records

You need to find a field that reload the whole page when changed and let it reload while auto filling form.

Join - https://efiller-doc.netlify.app/documentation/#join

**Reply by ryzugai** [reply]
i know nothing about the coding, you mean that i have to put delay for the next process right?

**Reply by jeevan-lal** [reply]
join whatsapp group and send me more details about this, like a video for how to fill this form manually
